### DISCUSSION TITLE
Water/ship related resources

### DISCUSSION THREAD

**Discussion by lshaf** [voxeltycoon/issues#356] [General] [answered]
- Ship of course
  - can moving resource via ship like passenger or other resource?
- Oil from water
- Fishing resource
- Drinking water?
- Shells
- Farm in water maybe?

**Comment by victorleblais** [comment, marked as answer]
Ships are planned. But I'm not sure how much of the other things you have listed are planned.

**Comment by lshaf** [comment]
Thanks for correcting mate.
I don't know it was planned but when I look up for "water" keyword it doesn't come with any result 😁.

**Reply by victorleblais** [reply]
It's on the roadmap: https://voxeltycoon.xyz/roadmap
